packages/typography/src/views/lists/List.example.md

```diff
@@ -0,0 +1,202 @@
+The following example shows unordered lists. Unordered lists may receive
+style types of `circle`, `disc`, or `square`.
+
+```jsx
+<p style={{marginTop: 0}}>Type <Code>disc</Code> (default)</p>
+
+<List>
+  <List.Item>Item</List.Item>
+  <List.Item>Item</List.Item>
+  <List.Item>Item</List.Item>
+</List>
+
+<p>Type <Code>circle</Code></p>
+
+<List type="circle">
+  <List.Item>Item</List.Item>
+  <List.Item>Item</List.Item>
+  <List.Item>Item</List.Item>
+</List>
+
+<p>Type <Code>square</Code></p>
+
+<List type="square">
+  <List.Item>Item</List.Item>
+  <List.Item>Item</List.Item>
+  <List.Item>Item</List.Item>
+</List>
+```
+
+The following example shows ordered lists. Ordered lists may receive style
+types of `decimal`, `decimal-leading-zero`, `lower-alpha`, `upper-alpha`,
+`lower-roman`, or `upper-roman`.
+
+```jsx
+<p style={{marginTop: 0}}>Type <Code>decimal</Code> (default)</p>
+
+<List ordered>
+  <List.Item>One</List.Item>
+  <List.Item>Two</List.Item>
+  <List.Item>Three</List.Item>
+</List>
+
+<p>Type <Code>decimal-leading-zero</Code></p>
+
+<List ordered type="decimal-leading-zero">
+  <List.Item>One</List.Item>
+  <List.Item>Two</List.Item>
+  <List.Item>Three</List.Item>
+</List>
+
+<p>Type <Code>lower-alpha</Code></p>
+
+<List ordered type="lower-alpha">
+  <List.Item>One</List.Item>
+  <List.Item>Two</List.Item>
+  <List.Item>Three</List.Item>
+</List>
+
+<p>Type <Code>upper-alpha</Code></p>
+
+<List ordered type="upper-alpha">
+  <List.Item>One</List.Item>
+  <List.Item>Two</List.Item>
+  <List.Item>Three</List.Item>
+</List>
+
+<p>Type <Code>lower-roman</Code></p>
+
+<List ordered type="lower-roman">
+  <List.Item>One</List.Item>
+  <List.Item>Two</List.Item>
+  <List.Item>Three</List.Item>
+</List>
+
+<p>Type <Code>upper-roman</Code></p>
+
+<List ordered type="upper-roman">
+  <List.Item>One</List.Item>
+  <List.Item>Two</List.Item>
+  <List.Item>Three</List.Item>
+</List>
+```
+
+### Interactive Demo
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { Checkbox, Field, Input, Label } = require('@zendeskgarden/react-forms/src');
+const {
+  Dropdown,
+  Select,
+  Field: SelectField,
+  Label: SelectLabel,
+  Menu,
+  Item: MenuItem
+} = require('@zendeskgarden/react-dropdowns/src');
+const { Range, RangeField, Label: RangeLabel } = require('@zendeskgarden/react-ranges/src');
+
+initialState = {
+  length: 1,
+  levels: 1,
+  ordered: false,
+  size: List.defaultProps.size,
+  start: 1
+};
+
+const text = 'garden es bonus vobis proinde vos postulo essum magis kohlrabi welsh onion daikon amaranth tatsoi tomatillo melon azuki bean garlic beet greens corn soko endive gumbo gourd shallot courgette tatsoi pea sprouts fava bean collard greens dandelion okra wakame tomato cucumber earthnut pea peanut soko zucchini.'.split(
```

Comment:
small nitpick request for the dummy text, is there any way to populate the list items with different text? i’m seeing rivers and it could mess with my perception of spacing. it would be helpful to have each list item have their own text.